scripts/circleci/copy-docker-images-to-cache.sh

```diff
@@ -16,14 +16,27 @@
 # Script which copies Docker images created during test runs to directory which
 # is cached
 
-if [ -f "~/agent_image/scalyr-agent-testings-fpm_package-builder" ]; then
+
+if [ -f ~/agent_image/scalyr-agent-testings-fpm_package-builder ]; then
+    echo "Copy image 'scalyr-agent-testings-fpm_package-builder' to cache."
     cp  ~/agent_image/scalyr-agent-testings-fpm_package-builder ~/fpm_builder_cache/scalyr-agent-testings-fpm_package-builder
     rm ~/agent_image/scalyr-agent-testings-fpm_package-builder
+else
+  echo "Image 'scalyr-agent-testings-fpm_package-builder' is not found."
 fi
 
-if [ -f "~/agent_image/scalyr-agent-testings-monitor-base" ]; then
+if [ -f ~/agent_image/scalyr-agent-testings-monitor-base ]; then
+    echo "Copy image 'scalyr-agent-testings-monitor-base' to cache."
     cp  ~/agent_image/scalyr-agent-testings-monitor-base ~/monitors_builder_cache/scalyr-agent-testings-monitor-base
     rm ~/agent_image/scalyr-agent-testings-monitor-base
+else
+  echo "Image 'scalyr-agent-testings-monitor-base' is not found."
 fi
 
-cp ~/agent_image/* ~/agent_image_cache/
+# the agent distribution image name should match to this glob.
+if [ "$(ls -A ~/agent_image/scalyr-agent-testings-distribution-*-base)" ]; then
```

Comment:
Yeah this ensures there are no potential left-overs from other images.

I was also thinking on how to improve that, but really didn't have the time to get to it yet.

At some point we should probably also use constants for some of that stuff. And we also need to make sure we document that in our caching docs :)